src/common/redis_module/runtest.py

```diff
@@ -131,7 +131,7 @@ def testObjectTableAddAndRemove(self):
     self.redis.execute_command("RAY.OBJECT_TABLE_REMOVE", "object_id1", "manager_id2")
     response = self.redis.execute_command("RAY.OBJECT_TABLE_LOOKUP", "object_id1")
     self.assertEqual(set(response), set())
-    # Remove a manager from an empty set, and make sure we still have an empty set.
+    # Remove a manager from an empty set, and make sure we now have an empty set.
```

Comment:
I changed the API so that it returns `None` for no entry at all and an empty set for an empty entry.